rct229/data/ashrae_90_1_table_G3_6.json

```diff
@@ -0,0 +1,120 @@
+{
+    "building_exteriors":[
+        {
+            "building_exterior_type": "Uncovered parking lots and drives",
+            "w/ft^2": 0.15,
+            "w/ft": null,
+            "w/location": null
+        },
+        {
+            "building_exterior_type": "Walkway - narrow",
+            "w/ft^2": null,
+            "w/ft": 1.0,
+            "w/location": null
+        },
+        {
+            "building_exterior_type": "Walkway - wide",
+            "w/ft^2": 0.2,
+            "w/ft": null,
+            "w/location": null
+        },
+        {
+            "building_exterior_type": "Plaza Areas",
+            "w/ft^2": 0.2,
+            "w/ft": null,
+            "w/location": null
+        },
+        {
+            "building_exterior_type": "Special Feature Areas",
+            "w/ft^2": 0.2,
+            "w/ft": null,
+            "w/location": null
+        },
+        {
+            "building_exterior_type": "Stairways",
+            "w/ft^2": 1.0,
+            "w/ft": null,
+            "w/location": null
+        },
+        {
+            "building_exterior_type": "Main entrance door",
+            "w/ft^2": null,
+            "w/ft": 30.0,
+            "w/location": null
+        },
+        {
+            "building_exterior_type": "Other entrance or exit doors",
+            "w/ft^2": null,
+            "w/ft": 20.0,
+            "w/location": null
+        },
+        {
+            "building_exterior_type": "Exterior canopies",
+            "w/ft^2": 1.25,
+            "w/ft": null,
+            "w/location": null
+        },
+        {
+            "building_exterior_type": "Outdoor sales - open areas",
+            "w/ft^2": 0.5,
+            "w/ft": null,
+            "w/location": null
+        },
+        {
+            "building_exterior_type": "Street frontage",
+            "w/ft^2": null,
+            "w/ft": 20.0,
+            "w/location": null
+        },
+        {
+            "building_exterior_type": "Non-tradable facade",
+            "w/ft^2": 0.2,
+            "w/ft": 5.0,
+            "w/location": null
+        },
+        {
+            "building_exterior_type": "Building facade",
+            "w/ft^2": 0.2,
+            "w/ft": 5.0,
+            "w/location": null
+
+        },
+        {       
+            "building_exterior_type": "Automated teller machines",
+            "w/ft^2": null,
+            "w/ft": null,
+            "w/location": 360  
```

Comment:
Maybe at least this one should have a "w/device" property. Then 90 would be the value for this property and w/location would be 270.